include/internal/csv_reader.cpp

```diff
@@ -427,7 +427,7 @@ namespace csv {
 
     CONSTEXPR void CSVReader::handle_unicode_bom(csv::string_view& in) {
         if (!this->unicode_bom_scan) {
-            if (in[0] == 0xEF && in[1] == 0xBB && in[2] == 0xEF) {
+            if (in[0] == '\xEF' && in[1] == '\xBB' && in[2] == '\xBF') {            
```

Comment:
Comparing `char` to integer constant > 128 (like `0xEF`) is always false if `char` is signed on a platform.